recipes/conda-deps/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "conda_deps" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4ee16fd4a54a6508265fcbc9e3048c0154319efdd2293f6fa7710cb7ee4a4035
+
+build:
+  number: 0
+  skip: True  # [win or py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - conda_deps --help
+
+about:
+  home: https://github.com/cgat-developers/conda-deps
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Generate conda environment files from Python source code'
```

Comment:
Nice! There may be some overlap with https://github.com/pyoceans/nbrr, let's talk and get the best of the two into one package.